agent/consul/client.go

```diff
@@ -288,15 +288,17 @@ func (c *Client) RPC(ctx context.Context, method string, args interface{}, reply
 	firstCheck := time.Now()
 	retryCount := 0
 	previousJitter := time.Duration(0)
+
+	metrics.IncrCounter([]string{"client", "rpc"}, 1)
 TRY:
 	retryCount++
 	manager, server := c.router.FindLANRoute()
 	if server == nil {
+		metrics.IncrCounterWithLabels([]string{"client", "rpc", "failed"}, 1, []metrics.Label{{Name: "server", Value: "nil"}})
```

Comment:
I'm not sure what the `Name` label is supposed to be used for here. Or whether I should be passing a different `Value` for the case of "error: no servers".

Reply:
This line is a copy of what's on L333 (just with the `Value` changed from `server.Name` to `"nil"`): https://github.com/hashicorp/consul/blob/8fe0bd1cbdfc743e67ba79555c1240b7fee36113/agent/consul/client.go#L331

Reply:
Best to use `IncrCounter` instead of the `...WithLabels` version since you don't actually have a server name to annotate with. Below they use that to indicate using a specific server has failed.